### PR TITLE
adds AVIF post

### PR DESCRIPTION
**Pull request by rachelandrew**
Fixes: #5477

### CODE REVIEW

File: src/site/content/en/blog/compress-images-avif/index.md
Comment (surma) [inline src/site/content/en/blog/compress-images-avif/index.md:60]: This image is unnecessarily bad and misrepresents JPEG. Please hold off on publishing this until I have reached out to the authors (but don’t let me block you on any other worker).

**Comment by jpmedley** [comment]
@rachelandrew Do you need me to do anything on this? Review it? Approve it? Jai is someone I usually suypport.

File: src/site/content/en/blog/compress-images-avif/index.md
Comment (wantehchang) [inline src/site/content/en/blog/compress-images-avif/index.md:249]: Please add "--" before "max 63".

File: src/site/content/en/blog/compress-images-avif/index.md
Comment (wantehchang) [inline src/site/content/en/blog/compress-images-avif/index.md:281]: Please change "0.63" to either "0..63" (two dots) or "0-63". This is intended to show the range of the quantize level, which is from 0 to 63, inclusive. Since we use "0-10" for the range of encoder speed at line 276, perhaps "0-63" is better?

**Comment by rachelandrew** [comment]
> @rachelandrew Do you need me to do anything on this? Review it? Approve it? Jai is someone I usually support.

I don't think so at the moment, re: Surma's comment. 

Rob asked me to work on this.

File: src/site/content/en/blog/compress-images-avif/index.md
Comment (wantehchang) [inline src/site/content/en/blog/compress-images-avif/index.md:284]: Nit: Should we change "think of cq-level is" to either

> think cq-level is

or

> think of cq-level as

?

File: src/site/content/en/blog/compress-images-avif/index.md
Comment (wantehchang) [inline src/site/content/en/blog/compress-images-avif/index.md:302]: Please change "quality level" to "quantize level".

File: src/site/content/en/blog/compress-images-avif/index.md
Comment (wantehchang) [inline src/site/content/en/blog/compress-images-avif/index.md:322]: Nit: It seems that we should remove the word "second", or perhaps change it to "first".

**Review by wantehchang — CHANGES_REQUESTED** [review]

**Review by wantehchang — CHANGES_REQUESTED** [review]

File: src/site/content/en/blog/compress-images-avif/index.md
Comment (wantehchang) [inline src/site/content/en/blog/compress-images-avif/index.md:180]: Nit: extensions => external dependencies

**Review by wantehchang — APPROVED** [review]
Thank you very much, Rachel. The only remaining change is the updated image comparing AVIF with JPEG.

**Comment by surma** [comment]
Spoke to the authors. Working on updating the images now :) Will let you know when I’m done.

**Review by surma — APPROVED** [review]
Images are fixed :)

**Review by wantehchang — APPROVED** [review]
Thank you, Surma!

File: src/site/content/en/blog/compress-images-avif/index.md
Comment (surma) [inline src/site/content/en/blog/compress-images-avif/index.md:49]:
```suggestion
  1120 x 840 AVIF at 18,769 bytes (click to enlarge)
```

File: src/site/content/en/blog/compress-images-avif/index.md
Comment (surma) [inline src/site/content/en/blog/compress-images-avif/index.md:56]:
```suggestion
  1120 x 840 JPEG at 20,036 bytes (click to enlarge)
```

**Comment by surma** [comment]
I sorted the CLA. I’ll leave the final call to merge to @robdodson

**Comment by robdodson** [comment]
I was able to build it locally to check and it looked good.

**Pull request merged**
